src/main/java/org/myrobotlab/service/Arduino.java

```diff
@@ -1943,10 +1968,15 @@ public void onServoMoveTo(ServoMove move) {
   // > servoSetVelocity/deviceId/b16 velocity
   public void onServoSetSpeed(ServoSpeed servoSpeed) {
 
+    if (!isConnected()) {

```

Comment:
Was this early return intended? It seems to be a nearly copy-paste of another early-return if-statement on line 1978, and so it will cause that later conditional to become dead code

Reply:
good eye - removed